src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/main.js

```diff
@@ -443,101 +469,49 @@ define(function() {
-                        url: '/admin/api/media/' + this.media.id + '?action=new-version',
-                        method: 'POST',
-                        paramName: 'fileVersion',
-                        showOverlay: false,
-                        skin: 'small',
-                        titleKey: 'sulu.upload.small-dropzone-title',
-                        instanceName: 'file-version-' + this.media.id,
-                        maxFiles: 1
-                    }
-                }
-            ]);
-        },
-
-        filesAddedHandler: function(newMedia) {
-            if (!!newMedia[0]) {
-                this.media = this.sandbox.util.extend(false, {}, this.media, newMedia[0]);
-                this.sandbox.emit('husky.overlay.media-edit.close');
-                this.editSingleMedia(this.media);
-                this.sandbox.emit('sulu.media.collections.save-media', this.media, this.savedCallback.bind(this), true);
-            }
-        },
-
-        /**
-         * Maps the overlay inputs back on a single model
+         * Saves the medias if form-data is valid and something was changed
          * @returns {Boolean} returns false if form is invalid, true if valid
          */
-        changeSingleModel: function() {
-            if (this.sandbox.form.validate(constants.infoFormSelector)) {
-                var data = this.sandbox.form.getData(constants.infoFormSelector),
-                    before = JSON.stringify(this.media);
-
-                this.media = this.sandbox.util.extend(false, {}, this.media, data);
-                
-                if (before !== JSON.stringify(this.media)) {
-                    this.sandbox.emit('sulu.media.collections.save-media', this.media, this.savedCallback.bind(this));
-                    this.media = null;
-                }
+        saveMultipleMedia: function() {
+            var promise = $.Deferred();
 
-                return true;
-            } else {
-                return false;
-            }
-        },
-
-        /**
-         * Maps the overlay input back on multiple models
-         * @returns {Boolean} returns false if form is invalid, true if valid
-         */
-        changeMultipleModel: function() {
+            // validate form data
             if (this.sandbox.form.validate(constants.multipleEditFormSelector)) {
-                var data = this.sandbox.form.getData(constants.multipleEditFormSelector),
-                    before = JSON.stringify(this.medias);
-
-                this.sandbox.util.foreach(this.medias, function(singleMedia, index) {
-                    this.medias[index] = this.sandbox.util.extend(false, {}, singleMedia, data.records[index]);
+                var formData = this.sandbox.form.getData(constants.multipleEditFormSelector),
+                    requests = [];
+
+                // loop through each media
+                this.sandbox.util.foreach(this.medias, function(currentMedia, index) {
```

Comment:
@nnatter have you thought about writing a method in the manager-service for saving multiple media?